### PR TITLE
Nicify landing page

### LINKED ISSUE #432: Nicify landing page for public consumption
- Youtube video showing MIES in action
- Add images (Databrowser, PAPlot, wavebuilder)
- Installation instructions with images
- Picture of rig
- Links to publications

### PR DESCRIPTION
**Pull request by t-b**
See https://github.com/AllenInstitute/MIES/tree/feature/nicify-documentation for a preview of the readme.

I wanted to do that now as https://github.com/AllenInstitute/MIES/issues/634 is approaching fast.

Close #432.

### CODE REVIEW

**Comment by t-b** [comment]
@timjarsky The open tasks from https://github.com/AllenInstitute/MIES/issues/432 is the bandicam video of MIES in action and the picture of a rig. Is that something you can supply?

**Comment by t-b** [comment]
@timjarsky I'm moving the user documentation for the PA plot to a separate PR. So this should be ready to be reviewed.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+MIES is a sweep based data acquisition software package for intracellular
+electrophysiology (patch clamp). It offers top of its class flexibility and
+robustness for stimulus generation, data acquisition/evaluation and has
+proven to be a valuable tool in the literature \[[1](https://doi.org/10.1038/s41593-019-0417-0),
+[2](https://elifesciences.org/articles/37349)\].
```
Comment (timjarsky) [inline README.md:5]: MIES is a proven [1, 2] sweep based data acquisition software package for intracellular electrophysiology (patch clamp). It offers top of its class flexibility and robustness for stimulus generation, data acquisition, and analysis.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  [NWBv2-format](https://alleninstitute.github.io/MIES/IPNWB/index.html) and
+  read it back in
+- Run custom code during data acquisition for [Automatic Experiment
+  Control](https://alleninstitute.github.io/MIES/file/_m_i_e_s___analysis_functions_8ipf.html#file-mies-analysisfunctions-ipf)
+- Completely automate the experimental setup using [JSON configuration files](https://alleninstitute.github.io/MIES/file/_m_i_e_s___configuration_8ipf.html#file-mies-configuration-ipf)
+- Interact with MIES from other programming languages (Python, C++, Javascript, ...) using [ZeroMQ](https://alleninstitute.github.io/MIES/ZeroMQ-XOP-Readme.html).
+- Use the integrated [scripting
+  language](https://alleninstitute.github.io/MIES/SweepFormula.html#the-sweep-formula-module)
+  for convenient on-the-fly data evaluation
+- Fully backwards compatible with **every** earlier MIES version
```
Comment (timjarsky) [inline README.md:20]: - Acquire data on up to eight headstage on a single DAC.
- Run up to five DACs in parallel.
- Create arbitrarily complex stimulus sets with an easy to use GUI
- Export all data, including all of its metadata, into the industry-standard NWBv2-format and read it back in
- Run custom code during data acquisition for Automatic Experiment Control
- Completely automate the experimental setup using JSON configuration files
- Interact with MIES from other programming languages (Python, C++, Javascript, ...) using ZeroMQ.
- Use the integrated scripting language for convenient on-the-fly data evaluation.
- Fully backward compatible with every earlier MIES version

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- Windows 10 x64
+- [Igor Pro 8.04 (nightly) or later](https://www.wavemetrics.com/)
+- [NIDAQ MX XOP](https://www.wavemetrics.com/products/nidaqtools/nidaqtools.htm)
+
+## Getting started
+
+- Download the latest release from [here](https://github.com/AllenInstitute/MIES/releases/tag/latest)
+- Run the installer
+- View the documentation
+  [here](https://alleninstitute.github.io/MIES/user.html)
```
Comment (timjarsky) [inline README.md:44]: - [Download](https://github.com/AllenInstitute/MIES/releases/tag/latest) the latest release
- Run the installer
- [View](https://alleninstitute.github.io/MIES/user.html) the documentation

File: README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Support statement
+
+The last released version receives fixes for all critical bugs.
+
+## Bug reporting
+
+Please have a look [here](https://alleninstitute.github.io/MIES/reportingbugs.html).
```
Comment (timjarsky) [inline README.md:52]: [Instructions](https://alleninstitute.github.io/MIES/reportingbugs.html)

File: Packages/doc/user.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+The MIES DataBrowser allows the user to view acquired sweeps and metadata
+during and after an experiment. It has several display modes that facilitate
+recording from multiple channels simultaneously.
+
+MIES is scalable: MIES can control up to 5 DACs (Instrutech/Heka and National
+Instruments), multiple amplifiers (Molecular Devices 700B), and pressure
+regulators (Proportion Air).
+
+MIES has two primary modes: The Test Pulse (TP) and Data Acquisition.
```
Comment (timjarsky) [inline Packages/doc/user.rst:36]: remove last line

**Comment by timjarsky** [comment]
@t-b instead of 'Loading past data' how about 'Flexible experimental data loading'

File: README.md (hunk 2 above)
Comment (timjarsky) [reply README.md:20]: @t-b we should mention something like:
- comprehensive experimental metadata acquisition and browsing tools

**Review by timjarsky — APPROVED** [review]

**Pull request merged**